internal/rbd/rbd_journal.go

```diff
@@ -568,7 +585,13 @@ func RegenerateJournal(
 	if rbdVol.JournalPool == "" {
 		rbdVol.JournalPool = rbdVol.Pool
 	}
-	volJournal = journal.NewCSIVolumeJournal("default")
+	// FIXME: this works only on Kubernetes, how do other CO supply metadata?
+	rbdVol.Owner, ok = volumeAttributes["csi.storage.k8s.io/pvc/namespace"]
+	if !ok {
+		util.DebugLog(ctx, "could not detect owner for %s", rbdVol.String())
+	}
```

Comment:
This should be Error message? check this one if the PV is encrypted?

Reply:
The code is from here, IMO it's better to keep it the same for being symmetrical between both mirrored and created PVC (for error msg & set owner only when PVC is encrypted ).  
https://github.com/ceph/ceph-csi/blob/ec6703ed583143bb0d7c654e55c3a5fcb5955526/internal/rbd/encryption.go#L255-L258

Reply:
Why do we need to print this debug log for all the PVC? even if you think it should be debug message, in the omap generator this should go as debug message if the PVC is encrypted. this avoids the unnecessary logging debug message for unencrypted PVC.

Reply:
>Why do we need to print this debug log for all the PVC? 

This is the current behaviour with cephcsi. 

>even if you think it should be debug message, in the omap generator this should go as debug message if the PVC is encrypted. this avoids the unnecessary logging debug message for unencrypted PVC.

I was thinking about keeping it very similar to PVC creation process.
(so that we don't have very different logic in omap generator and cephcsi )
I am fine with setting owner only if it is encrypted.

@nixpanic opinions?

Reply:
@Madhu-1 , debug logs are printed now only if `kmsID != ""`